crates/bevy_ecs/src/query/access.rs

```diff
@@ -504,15 +508,18 @@ impl<T: SparseSetIndex> Access<T> {
             conflicts.extend(self.component_read_and_writes.ones());
         }
         if self.reads_all_resources {
-            // QUESTION: How to handle `other.writes_all`?
+            if other.reads_all_resources {
+                return AccessConflicts::All;
+            }
             conflicts.extend(other.resource_writes.ones());
         }
 
         if other.reads_all_resources {
-            // QUESTION: How to handle `self.writes_all`.
+            if self.reads_all_resources {
+                return AccessConflicts::All;
+            }
             conflicts.extend(self.resource_writes.ones());
```

Comment:
These two new inner `if`s should check for `writes_all_resources` and not `reads_all_resources`.

Reply:
oh you'r right, that's a bug..
We don't have a unit test for this because there's nothing that queries all resources but not all components

Reply:
Should we remove the `AccessConflicts::ALL` conflicts for resources entirely?